tests/integration/helpers.py

```diff
@@ -108,7 +108,6 @@ async def build_and_deploy(
             apps=[database_app_name],
             status=status,
             raise_on_blocked=True,
-            raise_on_error=False,
```

Comment:
We ignored this due to early volume mount errors, but it will also hide other errors during start up.